src/go/k8s/kuttl-test.yaml

```diff
@@ -12,7 +12,9 @@ commands:
   - command: "kubectl taint nodes -l node-role.kubernetes.io/master= node-role.kubernetes.io/master:NoSchedule-"
   - command: "./hack/install-cert-manager.sh"
   - command: "kubectl create -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/e23ff77fceba6a5d9f190f5d1a123c87701dc964/bundle.yaml"
-  - command: "make deploy"
+  - command: "sh -c 'cd config/manager && kustomize edit set image vectorized/redpanda-operator=localhost/redpanda-operator:dev'"
+  - command: "sh -c 'kustomize build config/default | kubectl apply -f -'"
+  - command: "kind load docker-image localhost/redpanda:dev"
```

Comment:
Is there any reason to not use make deploy? Was there some problem?

Reply:
a PR in vtools (redpanda-data/vtools#551) is using taskfile instead of makefile, so removing the dependency with the makefile is what's being done here

Reply:
It will create friction maintaining 2 build scripts.

Reply:
would it be ok if we remove the makefile? the mix of taskfile/makefile was a bit confusing. since we have task for everything else. we can remove it, wdyt?

Reply:
I'm not sure if we would like to have any open source contributors. Contributors would not see build scripts nor CI runs. I will not strongly fight task over makefile or vice versa.